test/versions.test.js

```diff
@@ -36,14 +36,20 @@ describe('versions', () => {
   })
 
   it('handles alpha/beta releases', () => {
-    const versionInfo = getVersionInfo({
-      tag_name: 'v10.0.3-alpha',
-      name: 'Some release',
-    })
+    const versionInfo = getVersionInfo(
+      {
+        tag_name: 'v10.0.3-alpha',
+        name: 'Some release',
+      },
+      null,
+      'v10.0.3-alpha'
+    )
 
     expect(versionInfo.$NEXT_MAJOR_VERSION.version).toEqual('11.0.0')
     expect(versionInfo.$NEXT_MINOR_VERSION.version).toEqual('10.1.0')
-    expect(versionInfo.$NEXT_PATCH_VERSION.version).toEqual('10.0.4')
+    expect(versionInfo.$NEXT_PATCH_VERSION.version).toEqual('10.0.3')
```

Comment:
This was previously wrong. The next patch version after `v10.0.3-alpha` is actually `10.0.3` not `10.0.4`, but it was becoming `10.0.4` before because `10.0.3-alpha` was incorrectly becoming `10.0.3` internally